src/app/pages/forgot-password/forgot-password.page.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { AuthService } from 'src/app/services/auth.service';
+import { LoadingController, NavController } from '@ionic/angular';
+import { ToastService } from 'src/app/services/toast.service';
+
+@Component({
+  selector: 'app-forgot-password',
+  templateUrl: './forgot-password.page.html',
+  styleUrls: ['./forgot-password.page.scss'],
+})
+export class ForgotPasswordPage implements OnInit {
+  public forgotPasswordForm: FormGroup;
```

Comment:
```suggestion

  public forgotPasswordForm: FormGroup;

```

Just add empty line above and below. This separation will make readability nicer, that's all.